libs/media-player/src/lib/shared/as-two-digit-string.ts

```diff
@@ -0,0 +1,19 @@
+import { isInteger, isNonNegativeFiniteNumber } from '@coscrad/validation-constraints';
+
+/**
+ *
+ * @param inputNumber non-negative finite number less than 100
+ * @returns string
+ */
+
+export const asTwoDigitString = (inputNumber: number) => {
+    if (!isNonNegativeFiniteNumber(inputNumber) || inputNumber > 99) {
```

Comment:
Don't you want to also check that the input is an integer? I don't see how you would turn a decimal into a two-digit string.